src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -241,17 +242,25 @@ func (m *Manager) processUpdates() {
 
 func (m *Manager) processCustomPolling() {
 	for {
-		mutate, waitTime, err := m.CustomPoll(m.getCfg())
+		_, waitTime, _ := m.OnDemandPoll()
+		time.Sleep(waitTime)
+	}
+}
+
+func (m *Manager) OnDemandPoll() (func(cfg Config) error, time.Duration, error) {
```

Comment:
Allows things to trigger a lookup outside of the polling schedule. Added a lock to make sure it never conflicts with the polling itself.